pkgs/development/libraries/wt/cmake.patch

```diff
@@ -0,0 +1,10 @@
+--- a/CMakeLists.txt	2016-07-13 14:27:26.000000000 +0200
```

Comment:
What is the origin of this patch, why is it required, and can/should it be upstreamed?